Tests/DataflowType/AbstractDataflowTypeTest.php

```diff
@@ -18,17 +18,8 @@ public function testProcess()
 
         $dataflowType = new class($label, $options, $values, $testCase) extends AbstractDataflowType
         {
-            private $label;
-            private $options;
-            private $values;
-            private $testCase;
-
-            public function __construct(string $label, array $options, array $values, TestCase $testCase)
+            public function __construct(private string $label, private array $options, private array $values, private TestCase $testCase)
```

Comment:
Mettre les propriété auto déclaré par les constructeurs en readonly
```suggestion
            public function __construct(private readonly string $label, private readonly array $options, private readonly array $values, private readonly TestCase $testCase)
```

Reply:
readonly est seulement pour php 8.1